### ISSUE TITLE
[Story] As a site owner, I can manage a site that is WCAG 2.1 conformant.

### ISSUE THREAD

**Issue by inspector508** [uiowa/uiowa#1437]
•	All content is located in uniquely-named landmarks
•	All interactions can be achieved using the keyboard
•	Landmarks observe role restrictions and requirements
•	Shared components use valid ARIA attributes and values
•	Sitewide components meet WCAG guidelines
•	Themes and styles meet WCAG guidelines for contrast, magnification, and focus
•	Authoring tools support the creation of accessible, WCAG-conformant content

## Actionable Items
- [x] role=alert only for dynamic alerts, page loaded should be role=region with aria-label https://github.com/uiowa/uiowa/issues/1437#issuecomment-624676739
- [x] UIOWA Bar: Search button doesn’t announce changes in aria-expanded value.
Recommend test to determine cause (Blockquotefocus moves to another element) https://github.com/uiowa/uiowa/issues/1437#issuecomment-634686264
- [x] https://github.com/uiowa/uiowa/issues/1669 Search button label
- [x] https://github.com/uiowa/uiowa/issues/1680 Video cookie/reduced motion.
- [x] uids - accordion - Add aria relationship attributes (aria-controls, arialabelledby)
- https://github.com/uiowa/uiowa/issues/1320 fontawesome icon role _(blocked)_

## Related Files:

[UIOWA - Accessibility Report and Additional Notes.docx](https://github.com/uiowa/uiowa/files/4690078/UIOWA.-.Accessibility.Report.and.Additional.Notes.docx)
[UIOWA.EDU.xlsx](https://github.com/uiowa/uiowa/files/4690080/UIOWA.EDU.xlsx)

**Comment by inspector508** [comment]
To clarify, this content was initially intended for the ITAccessibility website, when I realized that it might also reinforce some of our early conversations around accessibility for this project. My agile story skills remain clumsy, for which I apologize.

**Comment by joewhitsitt** [comment]
role=alert only for dynamic alerts, page loaded should be role=region with aria-label

**Comment by bspeare** [comment]
UIOWA Bar: Search button doesn’t announce changes in aria-expanded value.
Recommend test to determine cause (Blockquotefocus moves to another element)

**Further details:** 

I think the issues was that we thought the “expanded/collapsed” notification would toggle more like the Accordion widget.
 
In the first segment of the attached video (Home page): when the user returns to the Search button, they are notified that the widget is expanded. But when they toggle the button, they are not notified that the widget is collapsed.
 
In the second segment (accordions), the state toggles with each user event. See unlisted demo at https://www.youtube.com/watch?v=ff-N6nOHAsQ

**Comment by joewhitsitt** [comment]
Closing this one for now. Going to create new one for future siteimprove a11y issues
